cli_tools/common/utils/param/populator.go

```diff
@@ -70,6 +73,13 @@ func (p populator) PopulateMissingParameters(project *string, clientID string, z
 		}
 	}
 
+	if n, s, err := p.NetworkResolver.Resolve(*network, *subnet, *region, *project); err == nil {
```

Comment:
Minor: the first call that handles an error as well is doing `err != nil` while the last two here do `err == nil`. Not consistent and might lead to bugs in the future.

Reply:
I see -- I was matching the behavior at line 70, but I agree that its best to have all of the checks consistent in the function. Updated this conditional, and the one at line 70 to handle `err != nil` first.